src/Redux/services/auth.service.js

```diff
@@ -1,31 +1,30 @@
 /* eslint-disable arrow-body-style */
 import axios from 'axios';
 
-const API_URL = 'https://technoquiz-env.eba-33dpsiuk.ap-south-1.elasticbeanstalk.com/api/user/';
+const API_URL =
+  'https://technoquiz-env.eba-33dpsiuk.ap-south-1.elasticbeanstalk.com/api/user';
```

Comment:
Url String Should contain '/' at the end that's why auth was not working on your local